app/templates/index.js

```diff
@@ -0,0 +1,4 @@
+var h5bp = require('h5bp');
+
+var app = h5bp.createServer({ root: __dirname + '/public' });
+app.listen(3000);
```

Comment:
@ngryman Our [`.editorconfig`](https://github.com/h5bp/generator-server-configs/blob/master/.editorconfig#L8) states we want newlines at the end of a file. This file doesn't have one. 
![scaffold_a_basic_app_for_node___fetch_it_via_npm__by_ngryman_ _pull_request__5_ _h5bp_generator-server-configs](https://cloud.githubusercontent.com/assets/6025224/5887323/dd924e16-a3c9-11e4-9dd8-d37281cde679.png)